src/fuzzer-state.js

```diff
@@ -119,6 +129,13 @@ export default class FuzzerState {
         st.allowSuperProp = false;
       }
     }
+    if (isAsync) {
+      st.allowAwaitExpr = true;
+      st.allowAwaitIdentifier = false;
+    } else {
+      st.allowAwaitExpr = false;
```

Comment:
This is the default, so no need to set this, right? Though I'm always fine with being explicit.

Reply:
This state is a clone of the outer one, not fresh, so it does need to be set explicitly (for cases like `async function f(){ function g(){ await } }`).